lib/charms/haproxy/v0/haproxy_route.py

```diff
@@ -142,7 +142,7 @@ def _on_haproxy_route_data_available(self, event: EventBase) -> None:
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 2
+LIBPATCH = 3
```

Comment:
Is this change intended?

Reply:
yes because I forgot to do `charmcraft publish-libs` when I merged the patch version 2 so I pushed the current version of the lib and set the patch to 2 to align with charmhub